dcos/resource_dcos_package.go

```diff
@@ -647,7 +669,7 @@ func resourceDcosPackageRead(d *schema.ResourceData, meta interface{}) error {
 		// Carry the checksum from the previous ID
 		parts := strings.Split(d.Id(), ":")
 		if len(parts) < 3 {
-			d.SetId("")
+			d.SetId()
```

Comment:
SetId needs an argument

Reply:
Sure, sorry. Strange how this typo could happen.